File: Łatwe/Java/Proste dodawanie/ProsteDodawanie.java
import java.util.Scanner;

class ProsteDodawanie {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i = 0; i < t; i++) {
            int n = in.nextInt();
            int sum = 0;
            for (int j = 0; j < n; j++) {
                int number = in.nextInt();
                sum += number;
            }
            System.out.println(sum);
        }
    }
}

File: Łatwe/C++/Proste dodawanie/ProsteDodawanie.cpp
#include <iostream>

int main() {
    int t;
    std::cin >> t;
    for (int i = 0; i < t; i++) {
        int sum = 0;
        int n;
        std::cin >> n;
        for (int j = 0; j < n; j++) {
            int number;
            std::cin >> number;
            sum += number;
        }
        std::cout << sum << std::endl;
    }
}
